src/bandersnatch/mirror.py

```diff
@@ -654,9 +653,7 @@ def save_json_metadata(self, package_info: Dict, name: str) -> bool:
         # In 4.0 we move to normalized name only so want to overwrite older symlinks
         if self.json_pypi_symlink(name).exists():
             self.json_pypi_symlink(name).unlink()
-        self.json_pypi_symlink(name).symlink_to(
-            os.path.relpath(self.json_file(name), self.json_pypi_symlink(name).parent)
-        )
+        self.json_pypi_symlink(name).write_bytes(self.json_file(name).read_bytes())
```

Comment:
In this case , symlink is replaced with `write_bytes` and `read_bytes`, these are identical to copy files